apps/wing/src/commands/lsp.ts

```diff
@@ -53,7 +52,7 @@ export async function lsp() {
       }
     } catch (e) {
       // set status in ide
-      connection.sendDiagnostics({
+      void connection.sendDiagnostics({
```

Comment:
@MarkMcCulloh For caution sake I added `void` to these async calls in the LSP to avoid accidentally changing existing behavior, though I don't know if some of them should be awaited perhaps